lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -487,3 +546,39 @@ GLOW_ONNXIFI_LIBRARY_FUNCTION_WRAPPER(onnxReleaseGraph)(onnxGraph graph) {
 
   return ONNXIFI_STATUS_SUCCESS;
 }
+
+// TODO do we need to have one onnxExtensionFunctionPointer per backendId?
```

Comment:
@yinghai @zrphercule could you comment on this? I'm reading https://github.com/onnx/onnx/blob/822d8df0a2a32233c6022f50a158817a0f19bdc7/onnx/onnxifi_ext.h#L28-L29 but not sure what to make of it

Reply:
Yeah. Just one copy is fine.